mne/cov.py

```diff
@@ -51,8 +51,7 @@ def _check_covs_algebra(cov1, cov2):
 
 def _get_tslice(epochs, tmin, tmax):
     """get the slice."""
-    tstart, tend = None, None
-    mask = _time_mask(epochs.times, tmin, tmax, sfreq=epochs.info['sfreq'])
+    mask = _time_mask(epochs.times, tmin, tmax, strict=True)
```

Comment:
Now `cov` and `SourceEstimate` calculations are the two places where `strict=True` is used, and `strict=False` is used everywhere else? That doesn't seem consistent to me.